src/scripts/components/foundations/forms/SignForm.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
```

Comment:
I don't think something this specific belongs in `foundations/`. I think a `forms/` directory _outside_ of `foundations/` makes more sense. IMO, `foundations/` should be reserved for very basic building blocks not specific to any particular piece of functionality

Reply:
I totally agree, @bent0b0x.
They are foundations form elements: Input and SignForm. Both just styled-components with no functionality. The `forms/` directory It's just a way to organize those components inside `foundations/`.
On `components/widgets/forms/TextField.js` we got our input with some functionality.

On the same `foundations/` directory we got `buttons/`, `svgs/` and `base/`. 
I think we could just move `base/` directory (with some settings of animation, colors, sizes, typography and themes) to outside the components.

What you think?

Reply:
Yes, I think moving the settings for animations, colors, etc makes sense _outside_ of `components/`. Maybe in `constants`/ as `AnimationConstants.js`, `ColorsConstants.js`, etc.

And as for where `SignForm.js` should go, I think the only thing that made me question it was the name. Might it be possible for us to rename this just `Form` (if it is generic enough to use in other contexts)? The way I think about it is everything in `foundations/` could, theoretically, be pulled out and used as a UI component library to build other apps in the future; so its components should be fairly generic and not specific to any particular application logic.

Again, this part isn't really that important so let's not let it hold us up 😄